db/migrate/20170329230309_create_requested_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRequestedTopics < ActiveRecord::Migration[5.0]
+  def change
+    create_table :requested_topics do |t|
+      t.string :name, null: false
```

Comment:
Do you think `name` is the best thing we could go with?

```ruby
@requested_topic.name
```

```ruby
@topic_request.title
```

Maybe? `.subject`?

Naming is Hard :tm: